dnfdaemon-client/utils.cpp

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
Who use functions from this file?

Reply:
Oh, thanks. It's a leftover of copying microdnf structure to dnfdaemon-client.